src/olympia/applications/templates/applications/appversions.html

```diff
@@ -15,6 +15,10 @@
 allowed for these applications.
 {% endtrans %}</p>
 
+<p >{% trans version_url=_('https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/applications') %}
```

Comment:
I hate to nitpick, but this isn't the prettiest HTML. Could you indent the inner content (the line below this) and strip the space in the `<p>` tag? ^_^